File: 백준/Bronze/10809. 알파벳 찾기/알파벳 찾기.js
let input = require('fs').readFileSync(0, 'utf-8').trim();
const alphabet = 'abcdefghijklmnopqrstuvwxyz'.split('');

let res = [];
alphabet.map((char) => {
  res.push(`${input.indexOf(char)}`);
});
console.log(res.join(' '));

File: 백준/Bronze/10809. 알파벳 찾기/알파벳 찾기.swift
let s: String = readLine()!
let alphabet: String = "abcdefghijklmnopqrstuvwxyz"

var res: [String] = []
for i in alphabet {
  var idx: Int = -1
  
  if let sIdx = s.firstIndex(of: i) {
    idx = s.distance(from: s.startIndex, to: sIdx)
  }
  
  res.append(String(idx))
}
print(res.joined(separator: " "))